app/questionnaire/questionnaire_schema.py

```diff
@@ -130,6 +130,42 @@ def get_questions_for_block(block_json):
         for question_json in block_json.get('questions', []):
             yield question_json
 
+    @staticmethod
+    def is_summary_section(section):
+        """Summary section is a section with a single Summary group
+        """
+        if section['groups']:
```

Comment:
If we change validator to require at least 1 group for a section then we don't need this check.

Reply:
Good point 👍